src/sqltest/parts/dashboard/widgets/propertiesWidget.component.test.ts

```diff
@@ -100,7 +101,13 @@ suite('Dashboard Properties Widget Tests', () => {
 			assert.fail('Called console Error unexpectedly');
 		};
 
-		let testComponent = new PropertiesWidgetComponent(dashboardService.object, new TestChangeDetectorRef(), undefined, widgetConfig, consoleError);
+		const testLogService = new class extends TestLogService {
```

Comment:
Why have this be the only one that fails if error is called? If anything I feel like this should be its own helper class so other tests can use it (ErrorFailTestLogService or something) since I can see the value in doing the check in general

Reply:
Oh I see - it was passing in a custom consoleError before. That's fine but I still think it's useful enough to make into a general helper class and put it in utils

Reply:
This code was specifically using the console logging to define an error, since this is an internal function and therefore hard to directly test. This could probably be rewritten to be better easy to test.